public/javascript/main.js

```diff
@@ -1,57 +1,64 @@
+$(function(){
+
+$.ajax('/game/config', {
+    type: 'GET'
+}).then(function(results){
```

Comment:
this seems to work, but something seems off/ wrong with this like it might give us trouble in the future... But I am not sure how to change it at this time. But for now looks good if we see some loading issues or saving data acting funny this is where we might want to look.